example/example_scenarios.YAML

```diff
@@ -30,3 +30,20 @@ example_scenario_2:
   tags:
     - 'smoke'
     - 'example2'
+
+example_scenario_composite:
+  grasshopper_args:
+    users: 10
+    spawn_rate: 1
+    runtime: 200
+  child_scenarios:
+    - scenario_name: example_scenario_1
```

Comment:
which values for grasshopper_args are being used for the child scenario? those that come from the composite scenario or from the child scenario?

Reply:
the composite scenario